scripts/qa_crowbarsetup.sh

```diff
@@ -2594,6 +2594,7 @@ function custom_configuration
             if [[ $hacloud = 1 ]] ; then
                 echo "Warning: Monasca currently is not HA ready"
             fi
+            proposal_set_value monasca default "['attributes']['monasca']['tsdb']" "'$monasca_tsdb'"
```

Comment:
We need a default here for `monasca_tsdb`. This script might be called from outside jenkins

Reply:
also you need to check if that parameter is available (the script is also used for cloud8 where we do not have this switch)

Reply:
Now I'm applying this only for Cloud 9+.